pkg/sfu/buffer/rtpstats.go

```diff
@@ -866,7 +866,7 @@ func (r *RTPStats) maybeAdjustFirstPacketTime(ts uint32) {
 			"extStartTS", r.extStartTS,
 		)
 		if r.firstTime.Sub(firstTime) > firstPacketTimeAdjustThreshold {
-			r.logger.Infow("first packet time adjustment too big, ignoring",
+			r.logger.Debugw("first packet time adjustment too big, ignoring",
```

Comment:
Can we leave this at Infow? I think the next round of deploy should not be logging as much. Want to check if the condition happens. Code in preprod does not log this. This is getting logged due to latching onto padding only packet I think. Preprod does not do that and is not seeing these logs. Want to check if that is true across the board.